src/dotnet.ts

```diff
@@ -103,6 +104,37 @@ export async function runDotNetCommand(dotNetSdk: DotNetInfo, args: string[], co
 			resolve();
 		});
 
+		dotnet.on('error', err => {
+			reject(err);
+		});
+	});
+}
+
+export async function runDotNetGetOutputFilePath(args: string[]): Promise<string> {
```

Comment:
I don't think we need two methods here, the code is almost identical. Why won't you just pass a delegate for ```handleData```?
This function makes very little sense to be honest, as it tries to encapsulate the behavior (as name implies GetOutputFilePath), but it doesn't even restrict the build target name. I think you should have one generic function that takes one or two delegates, then you can have two exported wrapper functions that lock down the actual target that is being called and the way they process the output.

Reply:
Or you keep one generic function here and do all customization in ```projectHelper.ts```